<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1341</width>
     <height>881</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="feedLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>691</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>571</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="topMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="tPOLabel">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>T-PO-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="tPOLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="companyNameLabel">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CompanyName</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="companyNameLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dO_dateLabel">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DO_date</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="dO_dateLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="dO_numberLabel">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DO_number</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="dO_numberLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="poRetakeButton">
         <property name="text">
          <string>Retake</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="companyNameRetakeButton">
         <property name="text">
          <string>Retake</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="doDateRetakeButton">
         <property name="text">
          <string>Retake</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="doNumberRetakeButton">
         <property name="text">
          <string>Retake</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>730</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>insert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertButton">
       <property name="text">
        <string>next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>180</y>
      <width>571</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="itemVLayout" stretch="1,100">
     <item>
      <layout class="QHBoxLayout" name="itemDescriptionHLayout" stretch="4,1,1">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>28</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;item_part_number&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>7</width>
           <height>28</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ordered&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_2">
         <property name="minimumSize">
          <size>
           <width>7</width>
           <height>28</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;received&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="itemHorizontalLayout" stretch="4,1,1">
       <item>
        <layout class="QVBoxLayout" name="itemComboVLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QComboBox" name="comboBox0"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox1"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox2"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox3"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox4"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox5"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox6"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox7"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox8"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox9"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox10"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox11"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="receivedVLayout">
         <item>
          <widget class="QLineEdit" name="orderedEdit0"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit1"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit4"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit5"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit6"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit7"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit8"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit9"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit10"/>
         </item>
         <item>
          <widget class="QLineEdit" name="orderedEdit11"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="orderedVLayout">
         <item>
          <widget class="QLineEdit" name="receivedEdit0"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit1"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit4"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit5"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit6"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit7"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit8"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit9"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit10"/>
         </item>
         <item>
          <widget class="QLineEdit" name="receivedEdit11"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
